osbs/core.py

```diff
@@ -420,19 +420,31 @@ def logs(self, build_id, follow=False, build_json=None, wait_if_missing=False):
             return response.iter_lines()
         return response.content
 
-    def list_builds(self, build_config_id=None, field_selector=None):
+    def list_builds(self, build_config_id=None, koji_task_id=None,
+                    field_selector=None):
```

Comment:
This is for the internal method. The API method for the OSBS class, the one you actually call, preserves the order (new parameter goes at the end).